### DISCUSSION TITLE
oval handle size (for moving nodes in space)

### DISCUSSION THREAD

**Discussion by autoca** [freeplane/freeplane#630] [Help]
Hi all,

Is it possible to increase the size of the oval handle of a node (that is used for moving it and its branch in space)?

I did not find such an option but maybe I overlooked something.

MTIA and regards, a.

**Comment by euu2021** [comment]
I know that it's possible to change the size of the folding icon, but I don't know about the oval handle.

A workaround is changing the size of the nodes. The smaller the node, the bigger is the size of the handle in relation to it:

![image](https://user-images.githubusercontent.com/77707706/188290889-db10d9e2-9241-40ae-bfd0-8ecfc22c0a73.png)


![image](https://user-images.githubusercontent.com/77707706/188290872-ef063e45-e223-4ff6-abae-35930b3f2267.png)

**Reply by autoca** [reply]
thanks for the brainstorming
